cmd/influx/internal/is_character_device.go

```diff
@@ -0,0 +1,19 @@
+package internal
+
+import (
+	"io"
+	"os"
+)
+
+// IsCharacterDevice returns true if the supplied reader is a character device (a terminal)
+func IsCharacterDevice(reader io.Reader) bool {
```

Comment:
can you move this to the cmd/influx main pkg itself. Not seeing the reuse here

Reply:
moved to write